packages/react-frontend/src/HomePage.js

```diff
@@ -1,75 +1,69 @@
-            }}
-            href={'#' + val}
-        >
-            {val}
-        </a>
+            }}>
+                {val}
+        </Link>
     )
 }
 
-function FunctionBar() {
+function HomeBody() {
     return (
         <div>
-            <SearchBar />
-            <div
-                style={{
-                    float: 'right',
-                }}
-            >
-                <FunctionButton val="Sign In" />
-                <FunctionButton val="Sign Up" />
-            </div>
-            <div
-                style={{
-                    'background-color': '#e9e9e9',
-                    width: '100%',
-                }}
-            >
-                <FunctionButton val="Home" />
-                <FunctionButton val="Profile" />
-                <FunctionButton val="Post" />
-            </div>
-
-            <div>
-                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-                eiusmod tempo incididunt ut labore et dolore magna aliqua. Ut
-                enim ad minim veniam, quis nostrud exercitation ullamco laboris
-                nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor
-                in reprehenderit in voluptate velit esse cillum dolore eu fugiat
-                nulla pariatur. Excepteur sint occaecat cupidatat non proident,
-                sunt in culpa qui officia deserunt mollit anim id est laborum.
-            </div>
-        </div>
-    )
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+            eiusmod tempo incididunt ut labore et dolore magna aliqua. Ut
+            enim ad minim veniam, quis nostrud exercitation ullamco laboris
+            nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor
+            in reprehenderit in voluptate velit esse cillum dolore eu fugiat
+            nulla pariatur. Excepteur sint occaecat cupidatat non proident,
+            sunt in culpa qui officia deserunt mollit anim id est laborum.
+        </div>);
 }
 
 function Home(props) {
-    return <FunctionBar />
+    return <div>
+        <BrowserRouter>
+        <div
+        style={{
+            'background-color': '#e9e9e9',
+            width: '100%',
+        }}>
+            <FunctionButton val="Home" />
+            <FunctionButton val="Profile" />
+            <FunctionButton val="Post" />
+        </div>
+
+        <div style={{
+            float: 'right',
+        }}
+        >
+            <FunctionButton val="Sign In" />
+            <FunctionButton val="Sign Up" />
+        </div>
+        <Routes>
+            <Route path="Sign In" element={<SignIn/>} />
+            <Route path="Sign Up" element={props.SignUp} />
+            <Route path="Home" element={props.Home}/>
+            <Route path="" element={props.Home}/>
+            {/* <Route path="Profile" element={<Profile />} /> */}
+            <Route path="Post" element={props.Post} />
+        </Routes>
+    </BrowserRouter>
+</div>
```

Comment:
This all looks really good to me. Would there be a way to redirect a user back to the home page if they every put in an invalid link?